dist/profile/templates/kubernetes/resources/evergreen_gateway/configmap.yaml.erb

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
As far as I understand, these credentials are being inserted and stored unencrypted in Kubernetes. Shouldn't values like `@parameters['db_pass']` be stored in a `Secret`?

Reply:
True, It was a shortcut that I should have cleaned earlier